src/components/timeline/GlobalTrack.js

```diff
@@ -176,7 +176,10 @@ class GlobalTrackComponent extends PureComponent<Props> {
               onMouseDown: this._onLabelMouseDown,
             }}
           >
-            <h1 className="timelineTrackName">{trackName}</h1>
+            {/* replaced the <h1> with <button> for navigation */}
```

Comment:
The comment is not useful to a reader :-) In this case I don't think a comment would be useful so you can just remove it.

(same for the other file below of course)

Reply:
Done 👍